HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -2941,6 +2943,35 @@ def delete
 
     def check_for_errors
       errors = []
+
+      if not @weekday_heating_setpoints.nil?
+        vals = @weekday_heating_setpoints.split(', ').map { |i| i.to_f }
+        if vals.length != 24
+          errors << "Weekday Heating Setpoints (#{vals.length}) must be of length 24."
```

Comment:
Can we be more explicit with these error messages? How about "... must be a comma-separated list of 24 numbers."?